coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 83.44
+    "covered_percent": 27.78
```

Comment:
Looks like we didn't ruin `rspec`.

Reply:
I did, problem with me. Idfk what's happening with it.